src/test/java/plugins/DashboardViewPluginTest.java

```diff
@@ -355,11 +360,170 @@ private void checkImages(BufferedImage actualImage, BufferedImage testImage) {
+        final boolean filterBuildExecutors = true;
+        Node s = slave1.install(jenkins).get();
+        s.configure();
+        s.setLabels("test");
+        s.save();
+
+        FreeStyleJob job = jenkins.jobs.create();
+
+        DashboardView v = createDashboardView();
+        v.configure(() -> {
+            v.mainArea.setFilterBuildExecutors(filterBuildExecutors);
+        });
+        v.open();
+
+        final List<String> headers = v.buildExecutorStatus.getHeaders();
+        final List<String> executors = v.buildExecutorStatus.getExecutors();
+        assertThat(headers.size(), is(2));
+        assertThat(executors.size(), greaterThan(1));
+
+        job.configure(() -> {
+            job.setLabelExpression("test");
+        });
+        v.open();
+
+        final List<String> headers2 = v.buildExecutorStatus.getHeaders();
+        final List<String> executors2 = v.buildExecutorStatus.getExecutors();
+        // If only one node, the title header is not shown.
+        assertThat(headers2.size(), is(0));
+        assertThat(executors2.size(), is(1));
+    }
+
+    @Test
+    public void changeColumns() {
+
+        DashboardView v = createDashboardView();
+        v.configure(() -> {
+
+            v.columnsArea.removeAll();
+            v.columnsArea.add(ColumnsArea.Column.NAME);
+            v.columnsArea.add(ColumnsArea.Column.LAST_FAILURE);
+            v.dashboardPortlets.checkIncludeStdJobList(true);
+        });
+
+        createFreeStyleJob();
+        v.open();
+
+        final List<String> titles = v.projectStatus.getHeaders();
+        titles.remove(titles.size() - 1); // last is not a name
+
+        assertThat(titles, hasSize(2));
+
+        assertThat(titles.get(0), containsString(ColumnsArea.Column.NAME.getText()));
+        assertThat(titles.get(1), containsString(ColumnsArea.Column.LAST_FAILURE.getText()));
+    }
+
+    @Test
+    public void configureDashboardFilterOnlyActivatedJobs() {
+        DashboardView v = createDashboardView();
+        BuildStatisticsPortlet stats = v.addBottomPortlet(BuildStatisticsPortlet.class);
+        v.configure(() -> {
+            v.jobFilters.setStatusFilter(JobFiltersArea.StatusFilter.ENABLED);
+        });
+
+        final FreeStyleJob active = createFreeStyleJob();
+        final FreeStyleJob disabled = createFreeStyleJob();
+
+        buildSuccessfulJob(active);
+        buildSuccessfulJob(disabled);
+
+        v.open();
+        assertThat(stats.getNumberOfBuilds(JobType.TOTAL), is(2));
+        assertThat(stats.getNumberOfBuilds(JobType.DISABLED), is(0));
+
+        disabled.configure(disabled::disable);
+
+        v.open();
+        assertThat(stats.getNumberOfBuilds(JobType.TOTAL), is(1));
+        assertThat(stats.getNumberOfBuilds(JobType.DISABLED), is(0));
+    }
+
+    @Test
+    @Ignore("The statistics portlet shows only one job in total (the disabled one). it is shown as successful (1) but disabled(0)." +
```

Comment:
@uhafner would be awesome if you could check if I did misunderstand something regarding the statusfilter `DISABLED`